service/pokemon_service.go

```diff
@@ -0,0 +1,25 @@
+package service
+
+import (
+	"WizelineApi/domain/model"
+	"errors"
+)
+
+//Servicio utilizado para buscar por ID dentro de un slice del modelod e Pokemon
+func GetPokemonById(pokemons []model.Pokemon, id int) (model.Pokemon, error){
```

Comment:
For better readability, for this and other acronyms, change it to all upper case letters. 

`Id` -> `ID`
```suggestion
func GetPokemonByID(pokemons []model.Pokemon, id int) (model.Pokemon, error){
```